File: go/Basics/Variables.go
package main

import (
	"fmt"
)

func main() {
	var a string
	var b int
	var c bool

	fmt.Println(a)
	fmt.Println(b)
	fmt.Println(c)

	var d, e = 6, "Hello"
	f, g := 7, "World!"

	fmt.Println(d)
	fmt.Println(e)
	fmt.Println(f)
	fmt.Println(g)

	var (
		h int
		j int    = 1
		k string = "hello"
	)

	fmt.Println(h)
	fmt.Println(j)
	fmt.Println(k)
}

File: java/basics/Variables.java
public class Main {

  static int myNum = 5;
  static float myFloatNum = 5.99f;
  static char myChar = 'D';
  static boolean myBool = true;
  static String myText = "Hello";
  static final int myFinal = 15; //value is fixed

  //Constructor
  public Main(
    int myNumArg,
    String myTextArg,
    char myCharArg,
    boolean myBoolArg,
    float myFloatNumArg
  ) {
    myNum = myNumArg;
    myFloatNum = myFloatNumArg;
    myChar = myCharArg;
    myBool = myBoolArg;
    myText = myTextArg;
  }

  //Method
  static void printVars() {
    System.out.println("int: " + myNum);
    System.out.println("float: " + myFloatNum);
    System.out.println("char: " + myChar);
    System.out.println("boolean: " + myBool);
    System.out.println("String: " + myText);
    System.out.println("Final: " + myFinal);
  }

  public static void main(String[] args) {
    System.out.println("Datatypes");
    printVars();
  }
}
